docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_YOLO_From_Tensorflow.md

```diff
@@ -143,11 +143,24 @@ cd darkflow
 #### <a name="yolov1-v2-to-tf"></a>Convert DarkNet\* YOLOv1 or YOLOv2 Model to TensorFlow\*
 
 To convert YOLOv1 or YOLOv2 model to TensorFlow, go to the root directory of the cloned DarkFlow repository and run the following command:<br>
+For Yolo V1:
 ```sh
-python3 ./flow --model <path_to_model>/<model_name>.cfg --load <path_to_model>/<model_name>.weights --savepb
+python3 flow --model yolov1.cfg --load yolov1.weights --savepb
 ```
 
-If the model was successfully converted, you can find the `<model_name>.meta` and `<model_name>.pb` files
+For Yolo V2 with VOC dataset `--labels` argument should be specified and additional changes in the original 
+exporting script are required. In the file`darkflow/utils/loader.py` change line 121 
```

Comment:
Can we add a hash commit for which the instruction was tested?

Reply:
last changes in this file were done 4 years ago, so decided not to add hash.
This is the latest commit and on that this solution worked
https://github.com/thtrieu/darkflow/blob/b187c65630f9aa1bb8b809c33ec67c8cc5d60124/darkflow/utils/loader.py
Do you want me to add anyways?

Reply:
Yes. Please, keep in mind that we should add this hash commit always.

Reply:
done, please merge if everything is ok